packages/gatsby/index.d.ts

```diff
@@ -84,12 +85,12 @@ export const prefetchPathname: (path: string) => void
  * export default (props: IndexPageProps) => {
  *   ..
  */
-export type PageProps<
-  DataType = object,
-  PageContextType = object,
+export interface PageProps<
```

Comment:
I'm not sure why this whole file was formatted. Did formatting settings change recently?